the-name-card/src/components/EditProfile/EditProfile.tsx

```diff
@@ -0,0 +1,309 @@
+import { ChangeEvent, useRef, useState } from 'react';
+import { ChromePicker, ColorResult } from 'react-color';
+import OutsideClickHandler from 'react-outside-click-handler';
+import { IoMdArrowBack } from 'react-icons/io';
+import { FaRegAddressCard } from 'react-icons/fa';
+import {
+	IoColorPaletteOutline,
+	IoClose,
+	IoPersonOutline,
+} from 'react-icons/io5';
+import { LuImagePlus } from 'react-icons/lu';
+import { BiMessageDetail } from 'react-icons/bi';
+import {
+	MdOutlineWorkOutline,
+	MdOutlineEmail,
+	MdPhone,
+	MdDragIndicator,
+} from 'react-icons/md';
+import SortableList, { SortableItem, SortableKnob } from 'react-easy-sort';
+import arrayMove from 'array-move';
+
+import {
+	Dropdown,
+	DropdownItem,
+	WidgetInitialValue,
+	WidgetItem,
+	WidgetProp,
+	WidgetType,
+} from '..';
+import './EditProfile.scss';
+
+export const EditProfile = () => {
+	// banner stuff for future refactor
+	const [bannerColor, setBannerColor] = useState<string>('#10A5F5');
```

Comment:
```suggestion
	const [bannerColor, setBannerColor] = useState('#10A5F5');
```
unnecessary type declartaion